src/main/java/core/basesyntax/service/CsvFileWriterService.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvFileWriterService {
+
+    public CsvFileWriterService(String filename) {
+        File file = new File("src" + File.separator + "main"
+                + File.separator + "resources" + File.separator + filename);
```

Comment:
Let's take the name of a file with a relative path as input, otherwise we are very rigidly bound to the place where the file should be located, and for files from other folders, we will have to create a new method to process them